packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -147,6 +147,106 @@ describe('KeyringController', () => {
     });
   });
 
+  describe('addNewAccountForKeyring', () => {
+    describe('when accountCount is not provided', () => {
+      it('should add new account', async () => {
+        await withController(
+          async ({ controller, initialState, preferences }) => {
+            const [primaryKeyring] = controller.getKeyringsByType(
+              KeyringTypes.hd,
+            ) as Keyring<Json>[];
+            const addedAccountAddress =
```

Comment:
Nit: Typically we separate the test into sections separated by newlines (the sections being "arrange, act, assert" for the setup, code-under-test, and assertions respectively). These tests would be a bit easier to read if we had those newlines surrounding each of these calls under test